src/text-editor-presenter.coffee

```diff
@@ -169,8 +183,11 @@ class TextEditorPresenter
         lines: {}
         highlights: {}
         overlays: {}
-      gutter:
-        lineNumbers: {}
+      gutters:
+        sortedDescriptions: []
```

Comment:
I'd like to push this to a follow-up, because there's a ripple effect of changes from this.